src/test/java/org/cactoos/io/Md5DigestOfTest.java

```diff
@@ -72,10 +72,12 @@ public void checksumFromFile() throws Exception {
             "Can't calculate the file's MD5 checksum",
             new HexOf(
                 new Md5DigestOf(
-                    new InputOf(
-                        new ResourceOf(
-                            "org/cactoos/digest-calculation.txt"
-                        ).stream()
+                    new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above. text.asString() will be called twice, and second call will fail.

Reply:
@Iprogrammerr the same above.